crypto/err/openssl.txt

```diff
@@ -594,13 +594,16 @@ EC_F_OSSL_ECDSA_SIGN_SIG:249:ossl_ecdsa_sign_sig
 EC_F_OSSL_ECDSA_VERIFY_SIG:250:ossl_ecdsa_verify_sig
 EC_F_PKEY_ECD_CTRL:271:pkey_ecd_ctrl
 EC_F_PKEY_ECD_DIGESTSIGN:272:pkey_ecd_digestsign
```

Comment:
Should this identifier (and a few others) remain after you renamed the related methods ?

Reply:
Hmm...probably we need to do a big cleanup in one go of this kind of thing before we do the final release.